src/App.js

```diff
@@ -1,15 +1,23 @@
 import "./App.css";
 import HomePage from "./pages/HomePage";
 import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import ProductListingPage from "./pages/ProductListingPage";
+import Navbar from "./components/NavBar/Navbar";
+import Footer from "./components/Footer/Footer";
 function App() {
   return (
-    <Router>
-      <div className='App'>
-        <Routes>
-          <Route exact path='/' element={<HomePage />} />
-        </Routes>
-      </div>
-    </Router>
+    <>
```

Comment:
I think this is unnecessary here as we only have one child for this parent. Hence we could avoid this and make `<Router></Router>` as a parent. Yes, it won't make a difference in the DOM but I think this looks better.